pkgs/tools/security/sherlock/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv, lib, fetchFromGitHub, python3, makeWrapper }:
+let pyenv = python3.withPackages (pp: with pp; [
+      beautifulsoup4
+      certifi
+      colorama
+      lxml
+      pysocks
+      requests
+      requests-futures
+      soupsieve
+      stem
+      torrequest
+    ]);
+in
+stdenv.mkDerivation rec {
+  pname = "sherlock";
+  version = "0.14.0";
+  srcs = fetchFromGitHub {
+    owner = "sherlock-project";
+    repo = pname;
+    rev = "f8566960d461783558b7bcba5c818d9275de492a";
+    sha256 = "sha256-6jG/SmsiEL63EcBrx2fcQDYbmMCA+A7Jsc3E4f5NGts=";
+  };
+
+  buildInputs = [ pyenv makeWrapper ];
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/bin
+    mkdir -p $out/bin/resources
+    cp ./sherlock/*.py $out/bin/
+    cp --recursive ./sherlock/resources/ $out/bin/
```

Comment:
Shouldn't ressources be under share?

Reply:
Sherlock relies on resources to be located under the same directory as sherlock script. I can patch sherlock script file so that resources can go under share.
https://github.com/sherlock-project/sherlock/blob/d0177bcfb4acc569908b263a88e045c8f7bf5cca/sherlock/sherlock.py#L565

Reply:
I think we should do that. Having those files under bin is not like it should be.

Reply:
patch added.